[2023-03-30 09:48:12,109] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:48:12] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:49:22,265] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:49:22] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:49:26,652] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:49:26,654] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:49:26] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:52:10,232] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:52:10] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:52:14,824] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:52:14,825] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:52:14] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:52:46,441] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:52:46] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:52:50,378] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:52:50,379] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:52:50] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:52:55,077] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:52:55] "POST /dog HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:53:19,261] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:53:19] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:53:29,790] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:53:29] "POST /dog HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:54:30,376] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:54:30] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:54:34,720] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:54:34,722] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:54:34] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:54:43,086] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:54:43] "POST /dog HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:17,018] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:17] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:18,466] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:18] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:21,852] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:21] "OPTIONS /dog?id=5 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:21,892] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:21] "DELETE /dog?id=5 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:25,763] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:25] "OPTIONS /dog?id=4 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:25,773] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:25] "DELETE /dog?id=4 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:29,572] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:29] "OPTIONS /dog?id=3 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:29,583] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:29] "DELETE /dog?id=3 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:32,937] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:32] "OPTIONS /dog?id=2 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:32,949] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:32] "DELETE /dog?id=2 HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:38,833] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:56:38,834] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:38] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:40,589] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 1', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:56:40,590] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:40] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:43,808] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:43] "POST /dog HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:45,451] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 2', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:56:45,452] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:45] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:56:48,774] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:56:48] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:08,313] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:08] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:14,203] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:14] "POST /dog HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:15,767] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 3', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:57:15,768] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:15] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:17,200] WARNING add_dog() L53   Erro ao adicionar c√£o 'Dog 3', C√£o de mesmo nome j√° salvo na base :/ - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/rest_api/app.py L53  
[2023-03-30 09:57:17,201] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:17] "[31m[1mPOST /dog HTTP/1.1[0m" 409 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:17,825] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:17] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:26,436] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:26] "GET /dogs HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-03-30 09:57:31,748] INFO _log() L224  127.0.0.1 - - [30/Mar/2023 09:57:31] "[31m[1mPOST /dog HTTP/1.1[0m" 422 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
